src/Tracker/utils/index.js

```diff
@@ -0,0 +1,45 @@
+export const sendAnalytics = (payload) => {
+  const request = new XMLHttpRequest()
+  request.open('POST', 'https://analytics-sdk-dev.onfido.com/v1/import')
+  request.setRequestHeader('Content-Type', "application/json")
+  request.setRequestHeader('Authorization', "Basic M002Z3tQWURwWTlUanlDKDNRTHIqTDZOQ1hVY2FLZUU6")
+
+  request.onload = () => {
+    if (request.status === 200 || request.status === 201) {
+      console.log(request.response)
+    }
+    else {
+      console.log(request)
+    }
+  }
+  request.onerror = () => console.log(request)
+
+  request.send(payload)
+}
+
+export const formatAnalytics = (woopra, eventName, properties) =>
+  JSON.stringify({
+    batch: [{
+      anonymousId: woopra.cookie,
+      channel: "Web SDK",
+      context: {
+        app: {
+          name: woopra.instanceName,
+          namespace: woopra.options.domain
+        },
+        library: {
+          name: "analytics-js", //made up values, I don't know what this is
+          version: "0.0.0" //made up values, I don't know what this is
+        },
+        screen: {
+          height: 480, //made up values, it will break without
+          width: 320 //made up values, it will break without
+        },
+        ...woopra.visitorData
+      },
+      event: eventName,
+      properties,
+      timestamp: woopra.last_activity,
```

Comment:
Isn't this weird. Because woopra is not being used, do you think the `last_activity` value is correct?

Reply:
I'm not sure what you mean when you say "woopra is not being used". I'm using Woopra to track these events

Reply:
but you are never calling the `track` method in woopra library. Are you sure the `last_activity` field is being handled even with this limitation?

Seems weird to be relying on a library but then not really using it as we are supposed to.

Reply:
It does work, regardless of the fact that we are not calling track because it's just returning `new Date()` when you call `woopra. last_activity`
https://github.com/Woopra/browser-tracker/blob/master/src/tracker.js#L85

Reply:
note that code is when you first initialize it. Not when you send a new event.

makes sense?

Reply:
Sorry, wrong line.
https://github.com/Woopra/browser-tracker/blob/75e4357b15ddc5ccf35acd4caec4a6fd17b178e0/src/tracker.js#L180
There is an event listener listening for move movements which updates `last_activity` value